src/lib/spinel/radio_spinel.cpp

```diff
@@ -1605,43 +1600,6 @@ otError RadioSpinel::Transmit(otRadioFrame &aFrame)
 
     mTransmitFrame = &aFrame;
 
-#if OPENTHREAD_CONFIG_MAC_HEADER_IE_SUPPORT && OPENTHREAD_CONFIG_TIME_SYNC_ENABLE
```

Comment:
Since this block is removed, where is the calling of `otMacFrameUpdateTimeIe` on posix platform?